daffodil-schematron/src/main/resources/iso-schematron-xslt2/ExtractSchFromXSD-2.xsl

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<!--
+    Extract embedded Schematron schemas in W3C XML Schemas schemas    
```

Comment:
So this makes me think this implementation can handle schematron rules embedded in the DFDL schema, but I didn't see tests for this. 

That's ok for now, but this is very (very) desirable to centralize the schematron rules in the DFDL schema in many cases, so we should add tests for this to illustrate how it works, and if for some reason it does not, add a JIRA ticket for activating/fixing that feature.

Reply:
Work on this in a follow up PR, or is something needed in this one?

Reply:
Took a quick pass at it, doesnt seem like a quick patch... here is a reference http://www.topologi.com/resources/schtrn_xsd_paper.html

Reply:
Let's put it off then. No reason to hold this up.

Reply:
Sounds good.

I have a hacked working version.  Will follow up with draft PR and work on cleaning it up.

Reply:
#463